pkg/kubelet/kubelet_pods_test.go

```diff
@@ -2093,28 +2106,42 @@ func TestGetExec(t *testing.T) {
 	)
 	var (
 		podFullName = kubecontainer.GetPodFullName(podWithUIDNameNs(podUID, podName, podNamespace))
-		command     = []string{"ls"}
 	)
 
 	testcases := []struct {
 		description string
 		podFullName string
 		container   string
+		command     []string
 		expectError bool
 	}{{
 		description: "success case",
 		podFullName: podFullName,
 		container:   containerID,
+		command:     []string{"ls"},
 	}, {
 		description: "no such pod",
 		podFullName: "bar" + podFullName,
 		container:   containerID,
+		command:     []string{"ls"},
 		expectError: true,
 	}, {
 		description: "no such container",
 		podFullName: podFullName,
 		container:   "containerBar",
+		command:     []string{"ls"},
 		expectError: true,
+	}, {
+		description: "null exec command",
+		podFullName: podFullName,
+		container:   containerID,
+		expectError: false,
+	}, {
+		description: "multi exec commands",
+		podFullName: podFullName,
+		container:   containerID,
+		command:     []string{`"#@#%$^%!@_*("`, "ls", `\\ls`},
```

Comment:
I would expect this to error since these commands aren't valid... perhaps we could use something more plausible here? Like
```suggestion
		command:     []string{"bash", "-c", "ls"},
```